tensorflow/training/docker/1.15.4/py3/cu110/Dockerfile.gpu

```diff
@@ -1,105 +1,99 @@
-# Nvidia does not publish a TensorRT Runtime library for Ubuntu 18.04 with Cuda 10.1 support, so we stick with cuda 10.0.
```

Comment:
instead of moving this file to 1.15 cu110, can we add a new directory inside the 1.15 dir for cu11?

Reply:
Updated